tests/Configurator/DockerComposeConfiguratorTest.php

```diff
@@ -234,8 +234,6 @@ public function testConfigureFileWithExistingMarks()
       - db-data:/var/lib/postgresql/data:rw
       # You may use a bind-mounted host directory instead, so that it is harder to accidentally remove the volume and lose all your data!
       # - ./docker/db/data:/var/lib/postgresql/data:rw
-    ports:
-      - "5432:5432"
```

Comment:
Why have you removed the ports in the tests?

Reply:
We deleted them from the recipes but i can revert that

Reply:
Removing them from the recipes is a mistake IMHO. What we should do is only expose the port but not attach it to a specific port locally. So, I would use:

```
ports:
    - 5432
```

That allows the port to be exposed and a random port to be used locally (which can be picked up by the Symfony CLI Web Server -- https://symfony.com/doc/current/setup/symfony_server.html#docker-integration). That way, you can also have several projects using the same deps and running independently at the same time.

Reply:
Anyway we don’t have to sync these test files with the recipes.

Reply:
Sure thing, just wanted to reiterate my point of view on ports and Docker :)

Reply:
You're doing well because I misunderstood initially. I'm going to edit the PR symfony/recipes#413